### PR TITLE
fix/ui-improvement-slash-menu

### PR DESCRIPTION
**Pull request by Hitesh-gurnani**

### CODE REVIEW

File: apps/frontend/src/extensions/SlashCommand/group.ts [hunk 1]
```diff
@@ -3,6 +3,16 @@ import { Group } from "./types"
...
+      {
+        name: "taskList",
+        label: "Task List",
+        iconName: "ListTodo",
+        description: "Task list with todo items",
+        aliases: ["todo"],
+        action: (editor) => {
+          editor.chain().focus().toggleTaskList().run()
+        },
+      },
```
Comment (joaorceschini) [inline apps/frontend/src/extensions/SlashCommand/group.ts]: why change the order of the commands?

**Comment by deewakar-k** [comment]
@Hitesh-gurnani provide description about your changes

**Comment by Hitesh-gurnani** [comment]
- [ ] reduce gap between texts and frame, there's blank space
Solution - the 0th group index shouldn't have margin top 
`className={cn(
                "col-[1/-1] mx-2 mt-4 select-none text-[0.65rem] font-semibold uppercase tracking-wider text-neutral-500 first:mt-0.5",
                groupIndex == 0 && "mt-0"
              )}`


- [ ]  move todo menu up as it is used frquently
changed the order of commands as we needed to move todo up

- [ ]  update bg color as per new ui/bg color and same stroke as dock: 3A3A3A
`dark:bg-background border-[#3A3A3A]` added this className and moved the className variable in Surface.tsx after the harcoded className so that if someone provided external className it should be able to override the existing ones

Fix: the todo shows 1,2 in bubble menu
Chnaged the lucide icon for this

**Review by deewakar-k — DISMISSED** [review]

**Comment by joaorceschini** [comment]
> * [ ]  reduce gap between texts and frame, there's blank space
>   Solution - the 0th group index shouldn't have margin top
>   `className={cn( "col-[1/-1] mx-2 mt-4 select-none text-[0.65rem] font-semibold uppercase tracking-wider text-neutral-500 first:mt-0.5", groupIndex == 0 && "mt-0" )}`
> * [ ]   move todo menu up as it is used frquently
>   changed the order of commands as we needed to move todo up
> * [ ]   update bg color as per new ui/bg color and same stroke as dock: 3A3A3A
>   `dark:bg-background border-[#3A3A3A]` added this className and moved the className variable in Surface.tsx after the harcoded className so that if someone provided external className it should be able to override the existing ones
> 
> Fix: the todo shows 1,2 in bubble menu Chnaged the lucide icon for this

nice, can you just return the order of the commands to how they were before please?
the rest was very good, thank you

**Review by joaorceschini — DISMISSED** [review]

**Comment by hiteshgurnani10-thera** [comment]
Sure will do it

**Comment by Hitesh-gurnani** [comment]
Sure will do it

**Comment by sajdakabir** [comment]
hey @Hitesh-gurnani , please take a pull from main branch

**Comment by sajdakabir** [comment]
hey @Hitesh-gurnani 
i got some issue in your pr please take a look into it–

1. for bg color please refer #136 issue details 
dark mode:
Screenshot 2024-09-27 at 7 18 48 PM

light mode:
Screenshot 2024-09-27 at 7 17 53 PM

2. also getting error in building, cmd to check build ```pnpm run build```
Screenshot 2024-09-27 at 7 21 58 PM

**Pull request closed without merging**